pkg/metadata/annotations.go

```diff
@@ -47,6 +48,7 @@ func IsInternalAnnotation(key string) bool {
 }
 
 // SetAnnotation sets the specified key, value annotation on the provided Unstructured object.
+// TODO(levi): This won't work for Pulumi-computed values. https://github.com/pulumi/pulumi-kubernetes/issues/826
```

Comment:
Hmm, should we fix this?

Reply:
I haven't seen any reports of users hitting this yet, and this bug has been present for a long time. I think it's safe to fix outside of this PR.

Reply:
https://github.com/pulumi/pulumi-kubernetes/issues/826